### DISCUSSION TITLE
Curso Fullstack .NET - Aula Autenticação via Cookie: Erro 400 na tentativa de fazer login.

### DISCUSSION THREAD

**Discussion by thiagobfo** [balta-io/forum#585] [Tira Dúvidas] [answered]
Até o passo da criação do usuário funcionou normalmente, mas na hora de autenticar via cookie estou recebendo sempre o erro 400.

![image](https://github.com/balta-io/forum/assets/104504600/52d897bd-609e-4fed-bd0a-26bace118ee6)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @thiagobfo, como vai?
Como está seu código de login? Seus requests? 

Tentou debugar para ver onde para?

**Reply by thiagobfo** [reply]
Bom dia balta @andrebaltieri ! Fui testar novamente e rodou, tinha uma "," num lugar errado kkkkk obrigado!

**Reply by andrebaltieri** [reply]
hahahahaha, acontece @thiagobfo!!! 🚀
